app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,7 @@
  *= require_tree .
  *= require_self
  */
+/* application.css */
```

Comment:
remove unnecessary comments

Reply:
Bhaiya, I removed this comment. 
Thanks you!

Reply:
its still here 👀

Reply:
I reverted one commit from this branch, that's why it exists until now. But now I have resolved it.